### DISCUSSION TITLE
localStorage inaccessible within jupyter html browser

### DISCUSSION THREAD

**Discussion by ClaytonAstrom** [pradyunsg/furo#523] [Potential Issue] [answered]
Hey!

Just a quick note - https://github.com/pradyunsg/furo/blob/c790c6d7a33f45856fa85670a4238a101efefd08/src/furo/theme/furo/base.html#L86 will throw when pages are opened in jupyterlab's html viewer - https://github.com/jupyterlab/jupyterlab/pull/5855 (or similar iframe environments)

![image](https://user-images.githubusercontent.com/3735885/188975592-1b3965eb-4baa-43a4-8760-4eae0448032f.png)

**Comment by pradyunsg** [comment, marked as answer]
Hmm... this theme's dark mode / light mode mechanisms have a baked-in assumption that it's being used in a regular non-sandboxed browser environment.
